core/blockchain.go

```diff
@@ -483,9 +484,11 @@ func (bc *BlockChain) dispatchTxUnindexer() {
 // This includes the following:
 // - transaction lookup indices
 // - updating the acceptor tip index
-func (bc *BlockChain) writeBlockAcceptedIndices(b *types.Block) error {
+func (bc *BlockChain) writeBlockAcceptedIndices(b *types.Block, skipTxIndices bool) error {
```

Comment:
nit: can we use `bc.cacheConfig.SkipTxIndexing` directly in this function instead of passing it through from the caller?

Reply:
This will likely shrink the diff quite a bit

Reply:
done